docs/source/learn/core_notebooks/dimensionality.ipynb

```diff
@@ -9,28 +9,29 @@
    },
```

Comment:
<p>Perhaps it would be useful to mention that batch dimensions always show up to the left <code>[batch2, batch1, support]</code>. Not sure how/where would it be better to mention it...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5746/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>Totally agree, you could add it here, when you show what happens with size. Or with shape using ellipsis.</p>

Reply:
<p>Added in a couple of places</p>